README.md

```diff
@@ -10,15 +10,9 @@ Access and stream your Twilio debugger logs along with your calls and messages.
 # Usage
 <!-- usage -->
```

Comment:
Drop the `usage` tags or oclif will just re-create this section. Example: https://github.com/twilio/plugin-debugger/commit/c7340c6c2edd19818fade58136be436f9c6afa73#diff-04c6e90faac2675aa89e2176d2eec7d8

Reply:
I removed the command from package.json, but I should do that too.

Reply:
Yeah, saw that after. Any reason to remove it?

Reply:
It's incorrect for plugins. Notice what it had for the installation instructions:
`npm install -g @twilio-labs/plugin-watch`

Reply:
Yup, but the rest is correct. Just dropping the `usage` tags and using the custom `usage` works.

Reply:
TIL... thanks for the tip